packages/teleport/src/Account/ManageDevices/AddDevice/AddDevice.tsx

```diff
@@ -200,6 +207,19 @@ export function AddDevice({
                   readonly={addDeviceAttempt.status === 'processing'}
                 />
               )}
+              {mfaOption.value === 'webauthn' && (
+                <FieldSelect
+                  width="50%"
+                  label="Allow Passwordless Login?"
+                  value={usageOption}
+                  options={deviceUsageOpts}
+                  onChange={(o: DeviceusageOpt) => {
+                    validator.reset();
```

Comment:
What's the reason behind calling `validator.reset()` here?

Reply:
ah, it was a copy paste left over, removed (but in general, it is to clear any validation errors when users switches options, otherwise the validation error carries over to the switched input boxes that uses the same validator)